Compile-and-Flash-STMBoard-Guide/README.adoc

```diff
@@ -0,0 +1,52 @@
+== *Compiling and Flashing STM Board*
+
+----
+This has been tested on Linux Mint and is expected to work on other Debian based distros
+----
+
+All commands will look like `this`
+
+=== Setup
+. Go to UCSolarCarTeam repo >> Recruit Resources, and clone
+. Navigate to STM32F4Discovery-Resources and run the auto installer `sudo ./install.sh`
+. This will download STLink, gcc4mbed, and CubeMX2Makefile on your computer and add gcc4mbed to your path
+. To add CubeMX2Makefile and STLink to your path, go to terminal then go to your profile to edit the PATH environment variable. If you don't know how to use vim or nano, google it. They are quite straight forward to use. `nano ~/.profile` or `vim ~/.profile`
+. Add CubeMX2Makefile to your path `export PATH=$PATH:/opt/CubeMX2Makefile`
+. Add stlink to your path `export PATH=$PATH:/opt/stlink`
```

Comment:
Do you coyp these into your .profile? If you do it's not clear here, but also the install script should do this automatically.